packages/skygear-core/lib/cloud/index.js

```diff
@@ -421,7 +421,7 @@ export function configModule(moduleName, options) {
   const { includeme } = require(moduleName);
   if (includeme !== undefined) {
     const settings = {};
-    includeme(module.exports, settings);
+    return new Promise(resolve => resolve(includeme(module.exports, settings)));
```

Comment:
We are not going to promisfy the return config module. We just want the transport start after the promise resolved if it return a promise. so we should just return the value here: `return resolve(includeme(module.exports, settings));`